### PR TITLE
webui sharing public without skeleton

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Do not use skeleton files for ```webUISharingPublic```  to speed up CI
## Related Issue

Part of https://github.com/owncloud/QA/issues/622

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature [hunk 1]
```diff
@@ -42,92 +49,117 @@ Feature: Share by public link
   @skipOnINTERNETEXPLORER @skipOnMICROSOFTEDGE @issue-30392
   Scenario: mount public link
     Given using server "REMOTE"
-    And user "user2" has been created with default attributes and skeleton files
+    And user "user2" has been created with default attributes and without skeleton files
+    And user "user1" has created folder "/simple-folder"
```
Comment (dpakach) [inline tests/acceptance/features/webUISharingPublic/shareByPublicLink.feature]: Use server "LOCAL" to create `simple folder`

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@HariBhandari07 there are conflicts. Please rebase, resolve and push.

**Comment by DeepDiver1975** [comment]
conflicting

**Comment by HariBhandari07** [comment]
Backport on #35572

**Pull request merged**